ortools/linear_solver/linear_solver.h

```diff
@@ -727,6 +727,12 @@ class MPSolver {
       const std::vector<MPSolver::BasisStatus>& variable_statuses,
       const std::vector<MPSolver::BasisStatus>& constraint_statuses);
 
+  void SetStartingLpBasisInt(const std::vector<int>& variable_statuses,
```

Comment:
I do not like that.
You can recover a void* specific instance from the interface. Can be the solver ptr, to set the base.
I am also open to discussing the missing basis statuses and add them to the enum.

Reply:
what is the difference between the final basis and the one you get from the regular call ?

Reply:
We can use `MPVariable::basis_status` & `MPConstraint::basis_status`, but for performance reasons for large similar problems we would prefer to get 2 complete vectors in one go, without needing to loop on variables & constraints.

To give you an idea, we use OR-Tools in [Antares Simulator](https://antares-simulator.readthedocs.io/en/latest/). This project requires us to solve large & similar LP problems and performance is key (every simulation lasts 6 hours on 12 CPUs, ~72 CPU-hours).